app/src/test/kotlin/com/intive/picover/validators/TextValidatorTest.kt

```diff
@@ -85,20 +70,16 @@ class TextValidatorTest : ShouldSpec(
 					maxLength = param.maxLength
 				}
 
-				tested.validate(param.text) shouldBe param.validationStatus
-				tested.validate(param.text).isValid() shouldBe param.isValid
-				tested.validate(param.text).errorMessageId shouldBe param.errorMessageResId
```

Comment:
It is not in the scope of `TextValidatorTest` to test the error message or `isValid` function from `ValidationStatus`

Reply:
I would even get rid of future test for such fields like, string resources, icons, drawables, etc.